bin/yaml/go.yaml

```diff
@@ -10,6 +10,10 @@ compilers:
       dir: golang-{{name}}
       create_untar_dir: true
       strip: true
+      after_stage_script:
+        - "export GOROOT=$(pwd)/go"
```

Comment:
Note this won't affect any previously installed go by default. I will have to manually force-reinstall all the go langs and re-compress them